lib/plugins/aws/package/compile/events/apiGateway/lib/restApi.js

```diff
@@ -97,7 +97,7 @@ module.exports = {
       );
     }
 
-    if (apiGateway.minimumCompressionSize) {
+    if (apiGateway.minimumCompressionSize || apiGateway.minimumCompressionSize === 0) {
```

Comment:
I think we could rewrite it to be a bit simpler with `apiGateway.minimumCompressionSize != null`

Reply:
Absolutely, good call - thanks for the suggestion. 😄  I didn't consider the configuration validation at a higher level ensuring an integer.

Have updated the commit and done a force push.